### PR TITLE
CORTX-32661: Glossary for Logging requirement related terms

### PR DESCRIPTION
**Pull request by cdeshmukh**
Signed-off-by: Chetan S. Deshmukh <chetan.deshmukh@seagate.com>

## Description
Adding IEM term to CORTX Glossary for Logging related requirements

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds new functionality)
- [ ] Breaking change (bug fix or new feature that breaks existing functionality)
- [ ] Third-party dependency update
- [x] Documentation additions or improvements
- [ ] Code quality improvements to existing code or test additions/updates

## Applicable issues

- This change fixes an issue: NA
- This change is related to an issue: CORTX-32661

## How was this tested?

Its a documentation change, no code testing involved

## Checklist

- [ ] The change is tested and works locally.
- [ ] New or changed settings in the solution YAML are documented clearly in the README.md file.
- [ ] All commits are signed off and are in agreement with the [CORTX Community DCO and CLA policy](https://github.com/Seagate/cortx/blob/main/doc/dco_cla.md).

If this change requires newer CORTX or third party image versions:

- [ ] The `image` fields in [solution.example.yaml](../k8_cortx_cloud/solution.example.yaml) have been updated to use the required versions.
- [ ] The `appVersion` field of the [Helm chart](../charts/cortx/Chart.yaml) has been updated to use the new CORTX version.

If this change addresses a CORTX Jira issue:

- [ ] The title of the PR starts with the issue ID (e.g. `CORTX-XXXXX:`)

-----
[View rendered GLOSSARY.md](https://github.com/cdeshmukh/cortx-k8s/blob/patch-1/GLOSSARY.md)

### CODE REVIEW

File: GLOSSARY.md [hunk 1]
```diff
@@ -64,6 +64,10 @@ Block devices, HDDs, SDDs, or other types of storage devices addressable by `/de
 
 Commonly known as "the package manager for Kubernetes", [Helm](https://helm.sh/) is a CLI tool that builds and deploys Kubernetes applications through application packages known as Helm Charts.
 
+### IEM
+
+IEM stands for Interesting Event Messages, relating to events happening within the system, affecting normal system functionality and which may be useful for preliminary debugging.
```
Comment (osowski) [inline GLOSSARY.md:69]: I would want to add in some additional commentary acknowledging that "IEM" is a pretty common term throughout Cloud Computing et al (similar to the definition of "Node" below).

```suggestion
In the context of [CORTX](https://github.com/Seagate/cortx), IEM stands for Interesting Event Messages, relating to events happening within the system, affecting normal system functionality and which may be useful for preliminary debugging. 
```

**Comment by cdeshmukh** [comment]
thanks @osowski, am ok with the suggestion & merged your changes too

**Comment by osowski** [comment]
As of September 12th, 2022, https://github.com/Seagate/cortx-k8s/blob/integration/GLOSSARY.md has been relocated to https://github.com/Seagate/cortx/blob/main/GLOSSARY.md.

This PR is being closed for the cortx-k8s repository and should be re-opened against https://github.com/Seagate/cortx/blob/main/GLOSSARY.md in the https://github.com/Seagate/cortx repository.

**Comment by cdeshmukh** [comment]
Created new PR https://github.com/Seagate/cortx/pull/1649 to update in new Glossary

**Pull request closed without merging**